### DISCUSSION TITLE
Condensation on my water meter

### DISCUSSION THREAD

**Discussion by flaschetelefon** [jomjol/AI-on-the-edge-device#1735] [General]
Due to condensation on my water meter, I have the problem that it cannot be detected. 

I could wipe it off with a cloth every day, but I'm afraid that the alignment won't fit. And I will definitely forget it.

Does anyone have a good solution to the problem? 

![grafik](https://user-images.githubusercontent.com/59640221/210136881-4a525b97-1903-400a-bda2-5e953acb6d99.png)

**Comment by forrestfunk** [comment]
Sorry, klingt vielleicht blöd, aber - (anders) lüften? Haben das Problem teilweise auch, wenn auch lang nicht so schlimm. Das Wasser kondensiert ja wahrscheinlich, weil die Uhr kalt ist und die Raumluft im Verhältnis dazu warm+feucht. 
Hast du einen geschlossenen Aufbau (3D Druck) als Halterung? Mein System ist offen und somit kommt mehr Luft an das ganze.
